### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by rubelux**
@nobletom123 Hey Tom, Can you have a look for compromise keys or data before merging?

### CODE REVIEW

File: components/Lib/GrantBox.js [hunk 1]
```diff
@@ -0,0 +1,1025 @@
...
+} from '@material-ui/icons/'
+import { Query } from 'react-apollo'
+import LeadArtist from './LeadArtists'
+import VideoViewer from './VideoViewer'
+import Router from 'next/router'
+import * as moment from 'moment'
+import MembersOnlyPopUp from './MembersOnlyPopUp'
+import Link from './Link'
+const videoPlayIcon =
+    'https://s3.amazonaws.com/kaleidoscope-web-assets/video_ui/video_player.png'
```
Comment (nobletom123) [inline components/Lib/GrantBox.js:20]: using external var instead?

File: components/Lib/VideoChat/Conference.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useRef, useState } from 'react'
+import VoxeetSdkL from '../../../lib/VoxeetSdk'
+
+const settings = {
+    consumerKey: 'NWl2MWYwbm1tams5',
+    consumerSecret: 'Nmc3bzQ4MnJhcTd1dG5rOWphZ2Iwdm5hM2w=',
+    conferenceAlias: 'Sample2',
+}
```
Comment (nobletom123) [inline components/Lib/VideoChat/Conference.js:8]: should these be in external variables/process.env variables?

File: components/Membership/data.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+]
+
+export const moviePosters = [
+    'https://res.cloudinary.com/kaleidoscope/image/upload/q_auto/v1551348580/web-ui/01_Battlescar_standard.jpg',
+    'https://res.cloudinary.com/kaleidoscope/image/upload/q_auto/v1551348580/web-ui/02_SPHERES_standard.jpg',
+    'https://res.cloudinary.com/kaleidoscope/image/upload/q_auto/v1551348580/web-ui/03_OldFriend_standard.jpg',
+    'https://res.cloudinary.com/kaleidoscope/image/upload/q_auto/v1551348581/web-ui/04_Apex_standard.jpg',
+    'https://res.cloudinary.com/kaleidoscope/image/upload/q_auto/v1551348582/web-ui/05_Terminal03_standard.jpg',
+    'https://res.cloudinary.com/kaleidoscope/image/upload/q_auto/v1551348581/web-ui/06_ZeroDaysVR_standard.jpg',
+]
```
Comment (nobletom123) [inline components/Membership/data.js:30]: these direct links okay/

File: components/Navigation/Header.js [hunk 4]
```diff
@@ -0,0 +1,901 @@
...
+                                className={
+                                    isMobile
+                                        ? logotypeContainerMobile
+                                        : logotypeContainer
+                                }
+                            >
+                                <a href="/" as="/" id="logotype">
+                                    <img
+                                        alt="logotype"
+                                        src="https://res.cloudinary.com/kaleidoscope/image/upload/v1551440044/web-ui/kaleido_logotype.png"
```
Comment (nobletom123) [inline components/Navigation/Header.js:508]: should this be a raw link?

File: lib/GosquaredApi.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+        if (!window._gs) {
+            const startScript = function (g, s, q, r, d) {
+                r = g[r] =
+                    g[r] ||
+                    function () {
+                        return (r.q = r.q || []).push(arguments)
+                    }
+                d = s.createElement(q)
+                q = s.getElementsByTagName(q)[0]
+                d.src = '//d1l6p2sc9645hc.cloudfront.net/tracker.js'
```
Comment (nobletom123) [inline lib/GosquaredApi.js:87]: should this be a direct url?

File: lib/confirmEmail.js [hunk 6]
```diff
@@ -0,0 +1,112 @@
+import * as mandrill from 'mandrill-api'
+import { sentenceToCamelCase, removeTimeStampFromTitle } from './utils'
+
+const mandrill_client = new mandrill.Mandrill('RO9HTVu8FGkJ5Ba5wexhrg')
```
Comment (nobletom123) [inline lib/confirmEmail.js:4]: should this be public?

**Review by nobletom123 — COMMENTED** [review]
a few url's/keys found

File: components/Lib/GrantBox.js (hunk 1 above)
Comment (rubelux) [reply components/Lib/GrantBox.js:20]: that's just an icon link from a CDN

File: components/Lib/VideoChat/Conference.js (hunk 2 above)
Comment (rubelux) [reply components/Lib/VideoChat/Conference.js]: Forgot this, This was a old legacy test and the account is not being used anymore, I deleted this folder completed

File: components/Membership/data.js (hunk 3 above)
Comment (rubelux) [reply components/Membership/data.js:30]: yeap, these are assets from Cloudinary which is an open access CDN

File: components/Navigation/Header.js (hunk 4 above)
Comment (rubelux) [reply components/Navigation/Header.js:508]: same

File: lib/GosquaredApi.js (hunk 5 above)
Comment (rubelux) [reply lib/GosquaredApi.js:87]: This is a CDN link to the goSquare tracking library rather a private or specific link to us

File: lib/confirmEmail.js (hunk 6 above)
Comment (rubelux) [reply lib/confirmEmail.js]: good one, moved it to a process env

**Comment by rubelux** [comment]
@nobletom123 thanks mate for the good review,  can you have a look to it again?

**Review by nobletom123 — APPROVED** [review]

**Review by nobletom123 — APPROVED** [review]

**Review by nobletom123 — APPROVED** [review]

**Review by nobletom123 — APPROVED** [review]

**Review by nobletom123 — APPROVED** [review]

**Review by nobletom123 — APPROVED** [review]

**Review by nobletom123 — APPROVED** [review]

**Pull request merged**